static/src/javascripts-legacy/projects/commercial/modules/creatives/fabric-expanding-v1.js

```diff
@@ -50,52 +50,49 @@ define([
     FabricExpandingV1.prototype.updateBgPosition = function () {
         var that = this;
 
-        var scrollY = window.pageYOffset;
         var viewportHeight = bonzo.viewport().height;
-        var adSlotTop = this.$adSlot.offset().top;
+        var adSlotTop = this.adSlot.getBoundingClientRect().top;
 
         var adHeight = (this.isClosed) ? this.closedHeight : this.openedHeight;
-        var inViewB = ((scrollY + viewportHeight) > adSlotTop);
-        var inViewT = ((scrollY - (adHeight * 2)) < adSlotTop + 20);
-        var topCusp = (inViewT &&
-        ((scrollY + (viewportHeight * 0.4) - adHeight) > adSlotTop)) ?
+        var inViewB = viewportHeight > adSlotTop;
```

Comment:
@katebee These expressions are just a means to detect whether the component is in the viewport. Now, I know that
```
bonzo(adSlot).offset().top ≡ window.pageYOffset + adSlot.getBoundingClientRect().top
```
and so all I do in these changes is drop the superfluous `window.pageYOffset`.